docs/react/add-to-existing.md

```diff
@@ -0,0 +1,358 @@
+---
+title: Add to Existing React Project
+sidebar_label: Add to Existing
+---
+
+import DocsCard from '@components/global/DocsCard';
+import DocsCards from '@components/global/DocsCards';
+
+<head>
+  <title>Add Ionic React to Existing Project: Integration Guide</title>
+  <meta
+    name="description"
+    content="Learn how to add Ionic React to your existing React project. Step-by-step guide for integrating Ionic components and features into an existing React application."
+  />
+</head>
+
+This guide covers how to add Ionic React to an existing React project. If you're looking to start a new project from scratch, check out the [Ionic React Quickstart](/docs/react/quickstart.md) guide. For an overview of how Ionic React works with React, including version support and tooling, check out the [Ionic React Overview](/docs/react/overview.md).
+
+:::tip
```

Comment:
Should we note somewhere that this follows adding to a React app created with Vite?

Reply:
Yes, especially if it looks different if it wasn't created with Vite.